modules/apps/organizations/organizations-test/src/testIntegration/java/com/liferay/organizations/service/test/OrganizationLocalServiceTest.java

```diff
@@ -745,7 +745,8 @@ public void testOrganizationObjectValidationRule() throws Exception {
 
 		AssertUtils.assertFailure(
 			ModelListenerException.class,
-			ObjectValidationRuleEngineException.class.getName(),
+			ObjectValidationRuleEngineException.class.getName() +
```

Comment:
can you please transform also the ObjectValidationRuleEngineException.class.getName() in a string? (I don't like the concatenation here) - it's just a "code format", the logic is fine